projects/core/src/auth/http-interceptors/client-token.interceptor.ts

```diff
@@ -33,7 +31,10 @@ export class ClientTokenInterceptor implements HttpInterceptor {
     return this.getClientToken(request).pipe(
       take(1),
       switchMap((token: ClientToken) => {
-        if (token && request.url.indexOf(this.baseReqString) > -1) {
+        if (
+          token &&
```

Comment:
What about change `take(1)` into `first(Boolean)` and leave out the first condition?

Reply:
`first(Boolean)` will make it wait for the token if not present, which we probably don't want here